src/Session.php

```diff
@@ -1024,10 +1026,36 @@ public static function getRightNameForError(int $right): string
             UPDATENOTE => 'UPDATENOTE',
             UNLOCK => 'UNLOCK',
         ];
-        if (array_key_exists($right, $rights)) {
-            return $rights[$right];
+        // Close session and force the default language so the logged right name is standardized
+        session_write_close();
+        $current_lang = $_SESSION['glpilanguage'];
+        $_SESSION['glpilanguage'] = 'en_GB';
```

Comment:
Is this working ?
In `NotificationTemplate` class, the call to `Session::loadLanguage($language);` seems to be required to ensure that locale files are loaded.